polaris-tokens/scripts/toStyleSheet.ts

```diff
@@ -78,34 +34,23 @@ export function getCustomProperties(tokenGroup: TokenGroup) {
 /**
  * Concatenates the `keyframes` token-group into a single string.
  */
-export function getKeyframes(motion: TokenGroup) {
+export function getKeyframes(motion: MetaTokenGroup) {
   return Object.entries(motion)
     .filter(([token]) => token.startsWith('keyframes'))
     .map(([token, {value}]) => `@keyframes p-${token}${value}`)
     .join('');
 }
 
-export async function toStyleSheet(
-  tokens: Tokens,
-  osColorSchemes: OSColorSchemes,
-) {
+export async function toStyleSheet(tokens: MetaTokens) {
   if (!fs.existsSync(cssOutputDir)) {
     await fs.promises.mkdir(cssOutputDir, {recursive: true});
   }
   if (!fs.existsSync(sassOutputDir)) {
     await fs.promises.mkdir(sassOutputDir, {recursive: true});
   }
 
-  const staticCustomProperties = getStaticCustomProperties(tokens);
-  const colorSchemeDeclarations = getColorSchemeDeclarations(
-    'light',
-    tokens,
-    osColorSchemes,
-  );
-  const defaultDeclarations = `${colorSchemeDeclarations}${staticCustomProperties}`;
   const styles = `
-  :root{${defaultDeclarations}}
-  ${getColorSchemeRules(tokens, osColorSchemes)}
+  :root{${getStaticCustomProperties(tokens)}}
```

Comment:
```suggestion
  :root{color-scheme: light;${getStaticCustomProperties(tokens)}}
```

I think we should still provide a hint to the browser: https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme